### PR TITLE
Order items

### PR DESCRIPTION
**Pull request by aljesusg**
captura de pantalla 2017-03-26 a las 13 37 39

### CODE REVIEW

**Comment by aljesusg** [comment]
I added specs too

File: app/models/manageiq/providers/consumption/event_manager.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module ManageIQ::Providers::Consumption::EventManager
+
+
+  def self.ems_type
+    @ems_type ||= "event_manager".freeze
```
Comment (sergio-ocon) [inline app/models/manageiq/providers/consumption/event_manager.rb]: Shoudn't thi9s be "showback_event_maanger"?

Comment (sergio-ocon) [inline app/models/manageiq/providers/consumption/event_manager.rb]: Consumption::COnsumptionManageir?

File: spec/factories/consumption_showback_bucket.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
-  factory ManageIQ::Providers::Consumption::ShowbackBucket, parent: :showback_bucket, class: ManageIQ::Providers::Consumption::ShowbackBucket do
+  factory ManageIQ::Providers::Consumption::ShowbackManager::ShowbackBucket, parent: :showback_bucket, class: ManageIQ::Providers::Consumption::ShowbackManager::ShowbackBucket do
```
Comment (sergio-ocon) [inline spec/factories/consumption_showback_bucket.rb]: Consumption::ConsumptionManager

File: spec/factories/consumption_showback_event.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
-  factory ManageIQ::Providers::Consumption::ShowbackEvent, parent: :showback_event, class: ManageIQ::Providers::Consumption::ShowbackEvent do
+  factory ManageIQ::Providers::Consumption::ShowbackManager::ShowbackEvent, parent: :showback_event, class: ManageIQ::Providers::Consumption::ShowbackManager::ShowbackEvent do
```
Comment (sergio-ocon) [inline spec/factories/consumption_showback_event.rb]: ConsumptionManager

File: spec/models/manageiq/providers/consumption/event_manager_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+describe ManageIQ::Providers::Consumption::EventManager do
```
Comment (sergio-ocon) [inline spec/models/manageiq/providers/consumption/event_manager_spec.rb]: ConsumptionManager

**Pull request merged**